src/csrs/model/organisationalUnit.ts

```diff
@@ -59,4 +64,17 @@ export class OrganisationalUnit {
 		return (this.abbreviation && this.abbreviation !== '') ? `${this.name} (${this.abbreviation})` : this.name
 	}
 
+	insertAndSortDomain(domain: Domain) {
+		if (this.domains.find(d => d.domain === domain.domain) === undefined) {
```

Comment:
You can use `.includes()` to see if an element exists in an array

Reply:
`includes` compares stuff using ===, so even if the two domain objects being compared here are the same, they would actually be different as they're technically two different objects (one stored in the `this.domains` array and one the incoming object from the method).

I will however use `this.doesDomainExist` for this instead, which is doing the same thing and an oversight from me!